### DISCUSSION TITLE
How to use translation

### DISCUSSION THREAD

**Discussion by ShuiHuo** [TandoorRecipes/recipes#1393] [Q&A]
I have translated most of the Chinese (simplified) content in translate.tandoor.dev, now how do I apply these translations to my server?

<details>
    <summary></summary>
我已经在translate.tandoor.dev翻译了大部分中文（简体）的内容，现在我该如何将这些翻译应用到我的服务器上？
</details>

**Comment by smilerz** [comment]
The translations will be applied in an upcoming release.
